src/index.html

```diff
@@ -16,17 +16,53 @@
       href="style.css"
     />
   </head>
+
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="bar"
+      data-qa="big"
+      method="post"
+    >
+      <button
+        type="submit"
+        class="button"
+      >
```

Comment:
The `alt` attribute for images should describe the image content. 'submit' does not describe the image; it should describe the action or content of the image, such as 'Search icon'.